codegen/codegen.cabal

```diff
@@ -30,7 +30,7 @@ library
     , aeson >= 1.4.2.0
     , inline-c-cpp >= 0.3.0.1
     , megaparsec >= 7.0.4
-    , show-prettyprint >= 0.2.2
+    -- , show-prettyprint >= 0.2.2 && < 0.3
```

Comment:
This was a bit problematic for me and I kept getting this error no matter how I adjusted the version / transitive version bounds:
```
Resolving dependencies...
Build profile: -w ghc-8.6.5 -O1
In order, the following will be built (use -v for more details):
 - show-prettyprint-0.2.3 (lib) (requires download & build)
 - codegen-0.1.0.0 (lib) (configuration changed)
 - codegen-0.1.0.0 (exe:codegen-exe) (configuration changed)
Downloading  show-prettyprint-0.2.3
Downloaded   show-prettyprint-0.2.3
Starting     show-prettyprint-0.2.3 (lib)
Building     show-prettyprint-0.2.3 (lib)

Failed to build show-prettyprint-0.2.3.
Build log (
/home/stites/.cabal/logs/ghc-8.6.5/show-prettyprint-0.2.3-6201d669a868176b795ee79386469aeacac1f00cf2cde985d02c90f5105476be.log
):
Configuring library for show-prettyprint-0.2.3..
Preprocessing library for show-prettyprint-0.2.3..
Building library for show-prettyprint-0.2.3..
[1 of 3] Compiling Text.Show.Prettyprint.Internal ( src/Text/Show/Prettyprint/Internal.hs, dist/build/Text/Show/Prettyprint/Internal.o )
[2 of 3] Compiling Text.Show.Prettyprint.Diagnostic ( src/Text/Show/Prettyprint/Diagnostic.hs, dist/build/Text/Show/Prettyprint/Diagnostic.o )

src/Text/Show/Prettyprint/Diagnostic.hs:33:74: error:
    • Couldn't match expected type ‘OldAnsiPpr.Doc’
                  with actual type ‘Doc
                                      prettyprinter-ansi-terminal-1.1.1.2:Data.Text.Prettyprint.Doc.Render.Terminal.Internal.AnsiStyle’
      NB: ‘Doc’
            is defined in ‘Data.Text.Prettyprint.Doc.Internal’
                in package ‘prettyprinter-1.5.1’
          ‘OldAnsiPpr.Doc’
            is defined in ‘Text.PrettyPrint.ANSI.Leijen.Internal’
                in package ‘ansi-wl-pprint-0.6.9’
    • In the first argument of ‘OldAnsiPpr.plain’, namely ‘e’
      In the first argument of ‘show’, namely ‘(OldAnsiPpr.plain e)’
      In the second argument of ‘(<>)’, namely
        ‘show (OldAnsiPpr.plain e)’
   |
33 |     Failure ErrInfo{ _errDoc = e } -> "ERROR " <> show (OldAnsiPpr.plain e)
   |                                                                          ^
cabal: Failed to build show-prettyprint-0.2.3 (which is required by
exe:codegen-exe from codegen-0.1.0.0). See the build log above for details.
```